test/storage/garbage_collector_test.cpp

```diff
@@ -1331,16 +1345,16 @@ TEST_F(GarbageCollectorTests, DeleteUpdateTest1) {
   }
 }
 
+/*
+ * Consider performing GC on the version chain sorted from newest to oldest:
+ * DELETE
+ * U2
+ * txn1 <- active txn
```

Comment:
Consistency with the previous notation
```suggestion
 T1 <- active txn
```